### ISSUE TITLE
Android keyboard problem when using autoFocus

### ISSUE THREAD

**Issue by omerman** [react-native-community/react-native-tab-view#660]
Seems like TabView rendering cancels the effect of autoFocus on TextInput.

once TabView is rendered, it immediately dismisses the keyboard. Only on Android.

all it takes is an app with this code:

```
  <View>
    <TabView
      navigationState={{
        index: 0,
        routes: [{ key: '0' }, { key: '1' }],
      }}
      onIndexChange={() => {}}
      renderScene={() => (<View><Text>Hi</Text></View>)}
    />
    <TextInput autoFocus />
  </View>
```

help please?

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.

**Comment by omerman** [comment]
Works perfect :ok_hand:

Thanks!
